_posts/2019-11-04-functional-traits.md

```diff
@@ -47,14 +47,18 @@ The most common metric used in trait-based studies is the community weighted mea
 
 <strong><em>Where to start</em></strong>
 
-Enquist et al. (2015), Funk et al. (2017), Pérez-Harguindeguy et al. (2013), Violle et al. (2012)
+Enquist et al. (2015), Funk et al. (2017), Pérez-Harguindeguy et al. (2013), Violle et al. (2012), Wigley et al. (2021)
 
 <strong> </strong>
 <h4><strong>4.16.2 Special cases, emerging issues, and challenges</strong></h4>
+All ecosystems have different challenges, and hence the importance of different traits vary between ecosystems. Open ecosystems such as grasslands, savannas, open woodlands and shrublands are often exposed to disturbances such as herbivory, fires and drought. To understand how climate change affects these systems, several traits that were not present in the Pérez-Harguindeguy et al. (2013) handbook were useful, and Wigley et al. (2021) gather tham all in a handbook for trait measurements in open ecosystems.  
+
```

Comment:
"All ecosystems have different challenges..."
This is a bit unclear, are you referring to global change? Fire, drought or just in general to different processes that occur?

And for the second sentence I would maybe rather write something like, that there are a couple of traits that are specifically useful for these ecosystems, because they measure responses to drought etc. (and then give 1-2 examples), which were not presented in Perez...